tests/sensor_test.go

```diff
@@ -6,11 +6,11 @@ import (
 	mock "home-collect/tests/mocks"
 	"testing"
 
-	"github.com/stretchr/testify/assert"
+	. "github.com/smartystreets/goconvey/convey"
 	"gopkg.in/mgo.v2/bson"
 )
 
-func TestVerifyIntegrityUrlSensor(t *testing.T) {
+/*func TestVerifyIntegrityUrlSensor(t *testing.T) {
```

Comment:
@adrissonGalvao please remove this comments.

Reply:
fixed it